### ISSUE TITLE
[Bug] Cannot install @nxrocks/nx-flutter in PNPM

### ISSUE THREAD

**Issue by rubiagatra** [tinesoft/nxrocks#131]
### Plugin Name

@nxrocks/nx-flutter

### Plugin Version

4.0.2

### Nx Version

14.6.4

### Expected Behaviour

Install latest nx-flutter with no problems

### Actual Behaviour

```
➜ pnpm add --save-dev @nxrocks/nx-flutter@4.0.2
 WARN  deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
 WARN  deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
 ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @nxrocks/common@0.0.0-development

The latest release of @nxrocks/common is "1.2.2".

If you need the full list of all 7 published versions run "$ pnpm view @nxrocks/common versions".
Progress: resolved 1224, reused 1223, downloaded 1, added 0
````

### Steps to reproduce the behaviour

1. Create new workspace `pnpm create nx-workspace`
2. `pnpm add -D @nxrocks/nx-flutter@4.0.2`

**Comment by tinesoft** [comment]
Hi @rubiagatra 
Thanks for reporting this.
Under investigation on https://github.com/tinesoft/nxrocks/tree/fix/flutter-pnpm...

**Comment by tinesoft** [comment]
In the meantime, creating a `.npmrc` file in the root of your workspace, with `strict-peer-dependencies` set to `false` can help.

See https://pnpm.io/npmrc#strict-peer-dependencies for more info

**Comment by rubiagatra** [comment]
I still can't install it @tinesoft with your suggestion. For now I only using Flutter CLI manually

**Comment by tinesoft** [comment]
Update:

I fixed the bug after updating Nx to `v15.0.4`, but the issue is back again after update to recent version of Nx.
So until they properly fix the problem, you can use this [workaround](https://pnpm.io/errors#err_pnpm_peer_dep_issues):

in your `package.json`:
```
{
  "pnpm": {
    "peerDependencyRules": {
      "ignoreMissing": ["@nxrocks/common"]
    }
  }
}
```
